templates/AureliaSpa/ClientApp/boot.ts

```diff
@@ -0,0 +1,15 @@
+import {Aurelia} from 'aurelia-framework';
+import 'bootstrap/dist/css/bootstrap.css';
+import 'bootstrap';
+ 
+// comment out if you don't want a Promise polyfill (remove also from webpack.config.js)
+import * as Bluebird from  'bluebird';
+Bluebird.config({ warnings: false });
+
+export function configure(aurelia: Aurelia) {
+  aurelia.use
+    .standardConfiguration()
+    .developmentLogging();
```

Comment:
Can you adapt this so that the `developmentLogging` only happens from dev builds and not from prod builds?